### PR TITLE
[REL] project_issue_solutions: Migration version 11.0 (renamed to helpdesk_solutions)

### PR DESCRIPTION
**Pull request by zaoral**
This module was moved from ingadhoc/project to
ingadhoc/enterprise-extensions

### CODE REVIEW

File: helpdesk_solutions/models/helpdesk_solution.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    _name = 'helpdesk.solution'
+    _inherit = ['mail.thread']
+    _description = "Helpdesk Solution"
+    _order = 'name'
+
+    name = fields.Char(
+        required=True,
+    )
+    solution_description = fields.Html(
+        string='Solution Description',
```
Comment (nicomacr) [inline helpdesk_solutions/models/helpdesk_solution.py]: @zaoral remove string here

File: helpdesk_solutions/models/helpdesk_solution.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    )
+    ticket_description = fields.Html(
+        string='Issue Description',
```
Comment (nicomacr) [inline helpdesk_solutions/models/helpdesk_solution.py]: and this should be "Ticket Description"

File: helpdesk_solutions/models/helpdesk_ticket.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+class HelpdeskTicket(models.Model):
+
+    _inherit = 'helpdesk.ticket'
+
+    helpdesk_solution_id = fields.Many2one(
+        'helpdesk.solution',
+        string='Linked Solution',
+    )
+    ticket_description = fields.Html(
+        string='Issue Description',
```
Comment (nicomacr) [inline helpdesk_solutions/models/helpdesk_ticket.py]: here should be "Ticket Description"

**Comment by zaoral** [comment]
@nicomacr 
The changes you requested were applied. 

Best Regards

**Pull request merged**